src/pages/images/actions/forms/UploadImageForm.tsx

```diff
@@ -58,6 +58,12 @@ const UploadImageForm: FC<Props> = ({ close, project }) => {
     }
   };
 
+  const clearCache = () => {
```

Comment:
great catch!

I just noticed that `fetchImageList` called in `EditGroupPanel` and `PermissionSelector` is cached with `queryKeys.images` only and not the project. So there would be a niche bug there since the permissions selector will not update correctly. Can we add `project` to the query keys for those api calls as part of this PR?

Reply:
Yes, good catch updated this to encounter for those two as well.

Reply:
aah yes, my suggestion is wrong, permissions operate outside of the scope of a single project so the cache keys there are correct so the invalidation logic should be updated. Thanks for the change :+1: